docs/Developers/Newcomers_Guide.rst

```diff
@@ -196,6 +196,8 @@ After that, you can run coala by simply typing
 
 into your bash. This will analyze your code and help you fix it.
 
+To test the code of your forked repository, check out this link `Adding CI to you fork <http://api.coala.io/en/latest/Developers/Adding_CI.html>`_
+
```

Comment:
umm..."check out this [link](http://api.coala.io/en/latest/Developers/Adding_CI.html)"
 looks better to me instead of 
"check out this link [Adding CI to you fork](http://api.coala.io/en/latest/Developers/Adding_CI.htmll)"